### PR TITLE
Add book module and index

### PR DESCRIPTION
**Pull request by kemeirnld2**

### CODE REVIEW

File: static/script.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+               $('#pwd').attr('type', 'password');
+          }
+     });
+     $("#search").on('keyup', function () {
+          var search = $(this).val().toLowerCase();
+          $('#data tbody tr').filter(function () {
+               $(this).toggle($(this).text().toLowerCase().indexOf(search) > -1);
+          });
+     });
+});
```
Comment (maxwelbarno) [inline static/script.js]: check whitespace!

File: templates/add_book.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+               <span class="input-group-text"><i class="fa fa-copy"></i></span>
+          </div>
+          <input type="number" class="form-control" name='number' required min="1">
+     </div>
+     <textarea name="description" class="form-control mb-3" cols="30" rows="2" required
+          placeholder="Book Description"></textarea>
+     <button class='btn btn-lg btn-primary btn-block' type='submit'>Submit</button>
+</form>
+
+{% endblock %}
```
Comment (maxwelbarno) [inline templates/add_book.html]: check whitespace!

File: templates/index.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                    </div>
+                </div>
+            </tr>
+            {% endfor %}
+        </tbody>
+    </table>
+    {% endif %}
+</div>
+
+{% endblock %}
```
Comment (maxwelbarno) [inline templates/index.html]: check whitespace!

**Review by maxwelbarno — CHANGES_REQUESTED** [review]

File: templates/add_book.html (hunk 2 above)
Comment (kemeirnld2) [reply templates/add_book.html]: WIP

**Review by maxwelbarno — APPROVED** [review]
Good job @kemeirnld2 

Proceed!

**Pull request merged**